### PR TITLE
Add test for comments counter

### LINKED ISSUE #2: [3pt]  - Add tests for comments counter - Student B
Make sure that function for the comments counter is covered with unit tests using Jest.

### PR DESCRIPTION
**Pull request by aimalamiri**
## Add test for the comments section
1. Added test for comment counter
2. Moved the loadComments method from Modal.js to commentsLoader.js file

closes #2

### CODE REVIEW

File: tests/commetsCounter.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+describe('Comments test', () => {
+  it('Should insert and count the comments', () => {
+    document.body.innerHTML = `
+      <h3 class="mt-3 text-lg"><span id="modal-comments-count" class="font-bold"></span> comments</h3>
+      <ul id="modal-comments"></ul>
+    `;
+    loadComments(comments);
+    const commentsCount = document.querySelector('#modal-comments-count').textContent;
+    expect(comments.length).toBe(Number(commentsCount));
```
Comment (rbanos-mv) [inline tests/commetsCounter.test.js]: In the general case of an equality (i.e. `a = b`)  `a` and `b` can be interchanged according to the _Commutative Property_. But for the test to be clear, it would be more appropriate that you expect the _result of your test_ `toBe` _a well-known value._ Please consider interchanging the `Number(commentsCount)` to the `expect()` and `comments.length` to the `toBe()`.

Have you considered what would happened in case `comments = []`? That could be another interesting case to test.

**Review by rbanos-mv — APPROVED** [review]
Congratulations. Well done!
You may consider the given suggestion before merging the PR.

File: tests/commetsCounter.test.js (hunk 1 above)
Comment (aimalamiri) [reply tests/commetsCounter.test.js]: I'm now using `toEqual` for the test, and I also added another test to check for the empty array.
